src/ConsoleApp/consoleAppDemo.java

```diff
@@ -7,14 +7,40 @@
  * @author Louis
  *
  */
-public class consoleAppDemo {
 
+import java.util.Scanner;
+
```

Comment:
You should never have more than one empty line to divide logical groups. In other words, Lines 12-14 should just be one line.